content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -452,20 +452,30 @@ for more information.
 
 ## pod-security.kubernetes.io/exempt
 
+
 Example: `pod-security.kubernetes.io/exempt: namespace`
 
-Used on: Event
+Used on: `audit.k8s.io/Event`
```

Comment:
Ok, I'll admit I was unsure about this one 😄 ! You are right, but it would be nice to have a standard way to know exactly which kind of resources these labels, annotations and taints apply on?